README.md

```diff
@@ -295,6 +295,7 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
  * [mos_game](https://github.com/bararchy/mos_game) - Mini Offline Singleplayer game
 
 ## GUI library
+ * [Cocoa.cr](https://github.com/Iainmon/Cocoa.cr) - A framework for developing macOS GUI applications using Crystal
```

Comment:
Let's omit `using Crystal` (it is implied here)

Reply:
@veelenga Sounds good! Should I make this change, and submit another pull request? I am fairly new to pull requests, so I am not sure what the best way to make this change would be.

Reply:
No, you can just make a new commit to this PR.

Reply:
@veelenga Just made the change!